recipes/metabat2/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "metabat2" %}
+{% set version = "2.12.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://bitbucket.org/berkeleylab/metabat/downloads/metabat-static-binary-linux-x64_v{{ version }}.tar.gz
+  md5: 4c8345ba5b3023b7b4cd16b38bd70c64
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build: 
+
+  run:
+
+
+test:
+  commands:
+    - metabat2 -h
```

Comment:
Please add more test for the other scripts as well. You are missing the Perl runtime dependency for example.